src/main/java/ch/bfh/bti7081/s2018/green/views/interfaces/IRetSetAbstField.java

```diff
@@ -0,0 +1,12 @@
+package ch.bfh.bti7081.s2018.green.views.interfaces;
+
+
+import com.vaadin.ui.AbstractField;
+
+public interface IRetSetAbstField {
```

Comment:
Why not just directly use `setValue` on the component? Why implement an interface?

Reply:
I need it a lot and directly want the component to be returned after setting a value. 
Sense: this should be a one-liner.

Maybe Lambda can help but I have no clue?